mimic/model/nova_objects.py

```diff
@@ -728,10 +747,12 @@ def request_ips(self, http_get_ips_request, server_id):
         return dumps({"addresses": server.addresses_json()})
 
     def request_list(self, http_get_request, include_details, absolutize_url,
-                     name=u"", limit=None, marker=None):
+                     name=u"", limit=None, marker=None, changes_since=None):
         """
         Request the list JSON for all servers.
 
+        :param str changes_since: ISO8601 formatted datetime
```

Comment:
This function even though currently used for changes-since should be independent. Will it help if the link is provided in `request_link` function?